client/package.json

```diff
@@ -34,10 +34,22 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint '*/**/*.{js,ts,tsx}' --fix"
+    "lint": "eslint '*/**/*.{js,ts,tsx}' --fix",
+    "storybook": "start-storybook -p 6006 -s public",
+    "build-storybook": "build-storybook -s public"
   },
   "eslintConfig": {
-    "extends": "react-app"
+    "extends": "react-app",
+    "overrides": [
```

Comment:
Not reviewing this, you are having way too many files changes for a simple schema ticket. Please fix this and then I will review. I recommend basing your branches properly instead of piling everything up. This makes reviewing processes slower and it's not valuable in a collaborative environment where time is important.